### DISCUSSION TITLE
ERROR invalid co header size

### DISCUSSION THREAD

**Discussion by PROGRAMADORRPL** [dashingsoft/pyarmor#1648] [General]
When I try to obfuscate my script, I run into the following error. Initially, I considered that it could be attributable to the restrictions inherent to the trial version; However, when purchasing the fully licensed version, the problem persists.

(base) roger@DESKTOP-OGF0FUV Modificaciones % pyarmor gen ventana_app.py
INFO     Python 3.11.5
INFO     Pyarmor 8.4.6 (basic), 00000, xxxxx
INFO     Platform darwin.x86_64

**Comment by jondy** [comment]
Duplicated issue #1647
